CONTRIBUTING.md

```diff
@@ -37,3 +37,20 @@ For example, for running tests in `multi`, `uuid` and `main` packages, call
 ```bash
 make test-multi test-uuid test-main
 ```
+
+To run benchmark tests, call
+```bash
+make bench DURATION=100x COUNT=7
```

Comment:
I would put a comment about possible values for DURATION and COUNT. I suppose COUNT accepts integer numbers, but with DURATION it is not so obvious (what `x` does mean?).